### ISSUE TITLE
[28] HTML5 error validation is not accessible to screen readers

### ISSUE THREAD

**Issue by bjgordon** [AusDTO/dta-website#301]

**Comment by bjgordon** [comment]
Report to recruiterbox.

**Comment by bjgordon** [comment]
Reported to recruiterbox. They fixed the issue on 29th March.
